pallets/registrar/src/lib.rs

```diff
@@ -853,6 +918,7 @@ pub mod pallet {
                 })?;
                 // Mark this para id for cleanup later
                 Self::schedule_parachain_cleanup(para_id)?;
+                T::InnerRegistrar::schedule_para_downgrade(para_id)?;
```

Comment:
let's add comments anywhere here, for all the actions, the PR is complex enough and I would prefer that everything is documented

Reply:
Docs added in 9a76e31422088f5a1231e8671063cb0ed3da1581 :)

Reply:
_fixed_